ERROR - 2022-12-29 09:20:09 --> Error connecting to the database: mysqli_sql_exception: Nenhuma conexão pôde ser feita porque a máquina de destino as recusou ativamente.
 in C:\xampp\htdocs\CI4M2\system\Database\MySQLi\Connection.php:171
Stack trace:
#0 C:\xampp\htdocs\CI4M2\system\Database\MySQLi\Connection.php(171): mysqli->real_connect('localhost', 'root', '', 'projeto-modulo-...', 3306, '', 0)
#1 C:\xampp\htdocs\CI4M2\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\CI4M2\system\Database\BaseConnection.php(561): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\CI4M2\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\CI4M2\system\Model.php(240): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\CI4M2\system\BaseModel.php(586): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\CI4M2\app\Controllers\Clientes.php(20): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\CI4M2\system\CodeIgniter.php(925): App\Controllers\Clientes->index()
#8 C:\xampp\htdocs\CI4M2\system\CodeIgniter.php(480): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Clientes))
#9 C:\xampp\htdocs\CI4M2\system\CodeIgniter.php(349): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\CI4M2\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\CI4M2\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Nenhuma conexão pôde ser feita porque a máquina de destino as recusou ativamente.
 in C:\xampp\htdocs\CI4M2\system\Database\MySQLi\Connection.php:215
Stack trace:
#0 C:\xampp\htdocs\CI4M2\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\CI4M2\system\Database\BaseConnection.php(561): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\CI4M2\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\CI4M2\system\Model.php(240): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\CI4M2\system\BaseModel.php(586): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\CI4M2\app\Controllers\Clientes.php(20): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\CI4M2\system\CodeIgniter.php(925): App\Controllers\Clientes->index()
#7 C:\xampp\htdocs\CI4M2\system\CodeIgniter.php(480): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Clientes))
#8 C:\xampp\htdocs\CI4M2\system\CodeIgniter.php(349): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\CI4M2\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\CI4M2\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#11 {main}
CRITICAL - 2022-12-29 09:20:09 --> Unable to connect to the database.
Main connection [MySQLi]: Nenhuma conexão pôde ser feita porque a máquina de destino as recusou ativamente.

in SYSTEMPATH\Database\BaseConnection.php on line 418.
 1 SYSTEMPATH\Database\BaseConnection.php(561): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `clientes`', [], false)
 3 SYSTEMPATH\Model.php(240): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(586): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\Clientes.php(20): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(925): App\Controllers\Clientes->index()
 7 SYSTEMPATH\CodeIgniter.php(480): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Clientes))
 8 SYSTEMPATH\CodeIgniter.php(349): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
ERROR - 2022-12-29 11:24:07 --> mysqli_sql_exception: Duplicate entry '4' for key 'PRIMARY' in C:\xampp\htdocs\CI4M2\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp\htdocs\CI4M2\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `fu...', 0)
#1 C:\xampp\htdocs\CI4M2\system\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `fu...')
#2 C:\xampp\htdocs\CI4M2\system\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `fu...')
#3 C:\xampp\htdocs\CI4M2\system\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `fu...', Array, false)
#4 C:\xampp\htdocs\CI4M2\system\Model.php(303): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\CI4M2\system\BaseModel.php(750): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\CI4M2\system\Model.php(695): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\CI4M2\app\Controllers\Funcionarios.php(63): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\CI4M2\system\CodeIgniter.php(925): App\Controllers\Funcionarios->store()
#9 C:\xampp\htdocs\CI4M2\system\CodeIgniter.php(480): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Funcionarios))
#10 C:\xampp\htdocs\CI4M2\system\CodeIgniter.php(349): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\CI4M2\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\CI4M2\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#13 {main}
CRITICAL - 2022-12-29 11:24:07 --> Duplicate entry '4' for key 'PRIMARY'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 292.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `funcionarios` (`nome`, `id_funcionario`, `rg`, `cpf`, `endereco`, `data_de_nascimento`, `telefone`, `data_de_contratacao`, `cargo`, `salario`, `dia_de_pagamento`) VALUES (\'Fulano de Tal2\', \'4\', \'123 546 789-7\', \'123\', \'0\', \'2022-12-02\', \'123\', \'2022-12-05\', \'inútil\', \'2000\', \'5\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `funcionarios` (`nome`, `id_funcionario`, `rg`, `cpf`, `endereco`, `data_de_nascimento`, `telefone`, `data_de_contratacao`, `cargo`, `salario`, `dia_de_pagamento`) VALUES (\'Fulano de Tal2\', \'4\', \'123 546 789-7\', \'123\', \'0\', \'2022-12-02\', \'123\', \'2022-12-05\', \'inútil\', \'2000\', \'5\')')
 3 SYSTEMPATH\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `funcionarios` (`nome`, `id_funcionario`, `rg`, `cpf`, `endereco`, `data_de_nascimento`, `telefone`, `data_de_contratacao`, `cargo`, `salario`, `dia_de_pagamento`) VALUES (\'Fulano de Tal2\', \'4\', \'123 546 789-7\', \'123\', \'0\', \'2022-12-02\', \'123\', \'2022-12-05\', \'inútil\', \'2000\', \'5\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `funcionarios` (`nome`, `id_funcionario`, `rg`, `cpf`, `endereco`, `data_de_nascimento`, `telefone`, `data_de_contratacao`, `cargo`, `salario`, `dia_de_pagamento`) VALUES (:nome:, :id_funcionario:, :rg:, :cpf:, :endereco:, :data_de_nascimento:, :telefone:, :data_de_contratacao:, :cargo:, :salario:, :dia_de_pagamento:)', [...], false)
 5 SYSTEMPATH\Model.php(303): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(750): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(695): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\Funcionarios.php(63): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(925): App\Controllers\Funcionarios->store()
10 SYSTEMPATH\CodeIgniter.php(480): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Funcionarios))
11 SYSTEMPATH\CodeIgniter.php(349): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
ERROR - 2022-12-29 11:24:37 --> mysqli_sql_exception: Duplicate entry '4' for key 'PRIMARY' in C:\xampp\htdocs\CI4M2\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp\htdocs\CI4M2\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `fu...', 0)
#1 C:\xampp\htdocs\CI4M2\system\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `fu...')
#2 C:\xampp\htdocs\CI4M2\system\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `fu...')
#3 C:\xampp\htdocs\CI4M2\system\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `fu...', Array, false)
#4 C:\xampp\htdocs\CI4M2\system\Model.php(303): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\CI4M2\system\BaseModel.php(750): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\CI4M2\system\Model.php(695): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\CI4M2\app\Controllers\Funcionarios.php(63): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\CI4M2\system\CodeIgniter.php(925): App\Controllers\Funcionarios->store()
#9 C:\xampp\htdocs\CI4M2\system\CodeIgniter.php(480): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Funcionarios))
#10 C:\xampp\htdocs\CI4M2\system\CodeIgniter.php(349): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\CI4M2\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\CI4M2\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#13 {main}
CRITICAL - 2022-12-29 11:24:37 --> Duplicate entry '4' for key 'PRIMARY'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 292.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `funcionarios` (`nome`, `id_funcionario`, `rg`, `cpf`, `endereco`, `data_de_nascimento`, `telefone`, `data_de_contratacao`, `cargo`, `salario`, `dia_de_pagamento`) VALUES (\'Fulano de Tal2\', \'4\', \'123 546 789-7\', \'123\', \'0\', \'2022-12-02\', \'123\', \'2022-12-05\', \'inútil\', \'2000\', \'5\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `funcionarios` (`nome`, `id_funcionario`, `rg`, `cpf`, `endereco`, `data_de_nascimento`, `telefone`, `data_de_contratacao`, `cargo`, `salario`, `dia_de_pagamento`) VALUES (\'Fulano de Tal2\', \'4\', \'123 546 789-7\', \'123\', \'0\', \'2022-12-02\', \'123\', \'2022-12-05\', \'inútil\', \'2000\', \'5\')')
 3 SYSTEMPATH\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `funcionarios` (`nome`, `id_funcionario`, `rg`, `cpf`, `endereco`, `data_de_nascimento`, `telefone`, `data_de_contratacao`, `cargo`, `salario`, `dia_de_pagamento`) VALUES (\'Fulano de Tal2\', \'4\', \'123 546 789-7\', \'123\', \'0\', \'2022-12-02\', \'123\', \'2022-12-05\', \'inútil\', \'2000\', \'5\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `funcionarios` (`nome`, `id_funcionario`, `rg`, `cpf`, `endereco`, `data_de_nascimento`, `telefone`, `data_de_contratacao`, `cargo`, `salario`, `dia_de_pagamento`) VALUES (:nome:, :id_funcionario:, :rg:, :cpf:, :endereco:, :data_de_nascimento:, :telefone:, :data_de_contratacao:, :cargo:, :salario:, :dia_de_pagamento:)', [...], false)
 5 SYSTEMPATH\Model.php(303): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(750): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(695): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\Funcionarios.php(63): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(925): App\Controllers\Funcionarios->store()
10 SYSTEMPATH\CodeIgniter.php(480): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Funcionarios))
11 SYSTEMPATH\CodeIgniter.php(349): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
ERROR - 2022-12-29 11:27:20 --> mysqli_sql_exception: Duplicate entry '4' for key 'PRIMARY' in C:\xampp\htdocs\CI4M2\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp\htdocs\CI4M2\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `fu...', 0)
#1 C:\xampp\htdocs\CI4M2\system\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `fu...')
#2 C:\xampp\htdocs\CI4M2\system\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `fu...')
#3 C:\xampp\htdocs\CI4M2\system\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `fu...', Array, false)
#4 C:\xampp\htdocs\CI4M2\system\Model.php(303): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\CI4M2\system\BaseModel.php(750): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\CI4M2\system\Model.php(695): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\CI4M2\app\Controllers\Funcionarios.php(63): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\CI4M2\system\CodeIgniter.php(925): App\Controllers\Funcionarios->store()
#9 C:\xampp\htdocs\CI4M2\system\CodeIgniter.php(480): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Funcionarios))
#10 C:\xampp\htdocs\CI4M2\system\CodeIgniter.php(349): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\CI4M2\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\CI4M2\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#13 {main}
CRITICAL - 2022-12-29 11:27:20 --> Duplicate entry '4' for key 'PRIMARY'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 292.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `funcionarios` (`nome`, `id_funcionario`, `rg`, `cpf`, `endereco`, `data_de_nascimento`, `telefone`, `data_de_contratacao`, `cargo`, `salario`, `dia_de_pagamento`) VALUES (\'Fulano de Tal2\', \'4\', \'123 546 789-7\', \'123\', \'0\', \'2022-12-02\', \'123\', \'2022-12-05\', \'inútil\', \'2000\', \'5\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `funcionarios` (`nome`, `id_funcionario`, `rg`, `cpf`, `endereco`, `data_de_nascimento`, `telefone`, `data_de_contratacao`, `cargo`, `salario`, `dia_de_pagamento`) VALUES (\'Fulano de Tal2\', \'4\', \'123 546 789-7\', \'123\', \'0\', \'2022-12-02\', \'123\', \'2022-12-05\', \'inútil\', \'2000\', \'5\')')
 3 SYSTEMPATH\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `funcionarios` (`nome`, `id_funcionario`, `rg`, `cpf`, `endereco`, `data_de_nascimento`, `telefone`, `data_de_contratacao`, `cargo`, `salario`, `dia_de_pagamento`) VALUES (\'Fulano de Tal2\', \'4\', \'123 546 789-7\', \'123\', \'0\', \'2022-12-02\', \'123\', \'2022-12-05\', \'inútil\', \'2000\', \'5\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `funcionarios` (`nome`, `id_funcionario`, `rg`, `cpf`, `endereco`, `data_de_nascimento`, `telefone`, `data_de_contratacao`, `cargo`, `salario`, `dia_de_pagamento`) VALUES (:nome:, :id_funcionario:, :rg:, :cpf:, :endereco:, :data_de_nascimento:, :telefone:, :data_de_contratacao:, :cargo:, :salario:, :dia_de_pagamento:)', [...], false)
 5 SYSTEMPATH\Model.php(303): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(750): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(695): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\Funcionarios.php(63): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(925): App\Controllers\Funcionarios->store()
10 SYSTEMPATH\CodeIgniter.php(480): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Funcionarios))
11 SYSTEMPATH\CodeIgniter.php(349): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
ERROR - 2022-12-29 11:27:57 --> mysqli_sql_exception: Duplicate entry '1' for key 'PRIMARY' in C:\xampp\htdocs\CI4M2\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp\htdocs\CI4M2\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `fu...', 0)
#1 C:\xampp\htdocs\CI4M2\system\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `fu...')
#2 C:\xampp\htdocs\CI4M2\system\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `fu...')
#3 C:\xampp\htdocs\CI4M2\system\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `fu...', Array, false)
#4 C:\xampp\htdocs\CI4M2\system\Model.php(303): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\CI4M2\system\BaseModel.php(750): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\CI4M2\system\Model.php(695): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\CI4M2\app\Controllers\Funcionarios.php(63): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\CI4M2\system\CodeIgniter.php(925): App\Controllers\Funcionarios->store()
#9 C:\xampp\htdocs\CI4M2\system\CodeIgniter.php(480): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Funcionarios))
#10 C:\xampp\htdocs\CI4M2\system\CodeIgniter.php(349): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\CI4M2\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\CI4M2\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#13 {main}
CRITICAL - 2022-12-29 11:27:57 --> Duplicate entry '1' for key 'PRIMARY'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 292.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `funcionarios` (`nome`, `rg`, `cpf`, `endereco`, `data_de_nascimento`, `telefone`, `data_de_contratacao`, `cargo`, `salario`, `dia_de_pagamento`, `id_funcionario`) VALUES (\'Fulano1\', \'123 546 789-7\', \'123\', \'0\', \'2022-12-02\', \'123\', \'2022-12-05\', \'inútil\', \'2000\', \'5\', \'1\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `funcionarios` (`nome`, `rg`, `cpf`, `endereco`, `data_de_nascimento`, `telefone`, `data_de_contratacao`, `cargo`, `salario`, `dia_de_pagamento`, `id_funcionario`) VALUES (\'Fulano1\', \'123 546 789-7\', \'123\', \'0\', \'2022-12-02\', \'123\', \'2022-12-05\', \'inútil\', \'2000\', \'5\', \'1\')')
 3 SYSTEMPATH\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `funcionarios` (`nome`, `rg`, `cpf`, `endereco`, `data_de_nascimento`, `telefone`, `data_de_contratacao`, `cargo`, `salario`, `dia_de_pagamento`, `id_funcionario`) VALUES (\'Fulano1\', \'123 546 789-7\', \'123\', \'0\', \'2022-12-02\', \'123\', \'2022-12-05\', \'inútil\', \'2000\', \'5\', \'1\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `funcionarios` (`nome`, `rg`, `cpf`, `endereco`, `data_de_nascimento`, `telefone`, `data_de_contratacao`, `cargo`, `salario`, `dia_de_pagamento`, `id_funcionario`) VALUES (:nome:, :rg:, :cpf:, :endereco:, :data_de_nascimento:, :telefone:, :data_de_contratacao:, :cargo:, :salario:, :dia_de_pagamento:, :id_funcionario:)', [...], false)
 5 SYSTEMPATH\Model.php(303): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(750): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(695): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\Funcionarios.php(63): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(925): App\Controllers\Funcionarios->store()
10 SYSTEMPATH\CodeIgniter.php(480): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Funcionarios))
11 SYSTEMPATH\CodeIgniter.php(349): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
ERROR - 2022-12-29 11:31:49 --> mysqli_sql_exception: Duplicate entry '4' for key 'PRIMARY' in C:\xampp\htdocs\CI4M2\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp\htdocs\CI4M2\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `fu...', 0)
#1 C:\xampp\htdocs\CI4M2\system\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `fu...')
#2 C:\xampp\htdocs\CI4M2\system\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `fu...')
#3 C:\xampp\htdocs\CI4M2\system\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `fu...', Array, false)
#4 C:\xampp\htdocs\CI4M2\system\Model.php(303): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\CI4M2\system\BaseModel.php(750): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\CI4M2\system\Model.php(695): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\CI4M2\app\Controllers\Funcionarios.php(63): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\CI4M2\system\CodeIgniter.php(925): App\Controllers\Funcionarios->store()
#9 C:\xampp\htdocs\CI4M2\system\CodeIgniter.php(480): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Funcionarios))
#10 C:\xampp\htdocs\CI4M2\system\CodeIgniter.php(349): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\CI4M2\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\CI4M2\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#13 {main}
CRITICAL - 2022-12-29 11:31:49 --> Duplicate entry '4' for key 'PRIMARY'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 292.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `funcionarios` (`nome`, `rg`, `cpf`, `endereco`, `data_de_nascimento`, `telefone`, `data_de_contratacao`, `cargo`, `salario`, `dia_de_pagamento`, `id_funcionario`) VALUES (\'Fulano de Tal3\', \'123 546 789-7\', \'123\', \'0\', \'2022-12-02\', \'123\', \'2022-12-05\', \'inútil\', \'2000\', \'5\', \'4\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `funcionarios` (`nome`, `rg`, `cpf`, `endereco`, `data_de_nascimento`, `telefone`, `data_de_contratacao`, `cargo`, `salario`, `dia_de_pagamento`, `id_funcionario`) VALUES (\'Fulano de Tal3\', \'123 546 789-7\', \'123\', \'0\', \'2022-12-02\', \'123\', \'2022-12-05\', \'inútil\', \'2000\', \'5\', \'4\')')
 3 SYSTEMPATH\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `funcionarios` (`nome`, `rg`, `cpf`, `endereco`, `data_de_nascimento`, `telefone`, `data_de_contratacao`, `cargo`, `salario`, `dia_de_pagamento`, `id_funcionario`) VALUES (\'Fulano de Tal3\', \'123 546 789-7\', \'123\', \'0\', \'2022-12-02\', \'123\', \'2022-12-05\', \'inútil\', \'2000\', \'5\', \'4\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `funcionarios` (`nome`, `rg`, `cpf`, `endereco`, `data_de_nascimento`, `telefone`, `data_de_contratacao`, `cargo`, `salario`, `dia_de_pagamento`, `id_funcionario`) VALUES (:nome:, :rg:, :cpf:, :endereco:, :data_de_nascimento:, :telefone:, :data_de_contratacao:, :cargo:, :salario:, :dia_de_pagamento:, :id_funcionario:)', [...], false)
 5 SYSTEMPATH\Model.php(303): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(750): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(695): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\Funcionarios.php(63): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(925): App\Controllers\Funcionarios->store()
10 SYSTEMPATH\CodeIgniter.php(480): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Funcionarios))
11 SYSTEMPATH\CodeIgniter.php(349): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
ERROR - 2022-12-29 11:37:25 --> mysqli_sql_exception: Duplicate entry '11' for key 'PRIMARY' in C:\xampp\htdocs\CI4M2\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp\htdocs\CI4M2\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `fu...', 0)
#1 C:\xampp\htdocs\CI4M2\system\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `fu...')
#2 C:\xampp\htdocs\CI4M2\system\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `fu...')
#3 C:\xampp\htdocs\CI4M2\system\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `fu...', Array, false)
#4 C:\xampp\htdocs\CI4M2\system\Model.php(303): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\CI4M2\system\BaseModel.php(750): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\CI4M2\system\Model.php(695): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\CI4M2\app\Controllers\Funcionarios.php(63): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\CI4M2\system\CodeIgniter.php(925): App\Controllers\Funcionarios->store()
#9 C:\xampp\htdocs\CI4M2\system\CodeIgniter.php(480): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Funcionarios))
#10 C:\xampp\htdocs\CI4M2\system\CodeIgniter.php(349): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\CI4M2\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\CI4M2\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#13 {main}
CRITICAL - 2022-12-29 11:37:25 --> Duplicate entry '11' for key 'PRIMARY'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 292.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `funcionarios` (`nome`, `rg`, `cpf`, `endereco`, `data_de_nascimento`, `telefone`, `data_de_contratacao`, `cargo`, `salario`, `dia_de_pagamento`, `id_funcionario`) VALUES (\'Fulano de Tal10\', \'123 546 789-7\', \'123\', \'0\', \'2022-12-02\', \'123\', \'2022-12-05\', \'inútil\', \'2000\', \'5\', \'11\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `funcionarios` (`nome`, `rg`, `cpf`, `endereco`, `data_de_nascimento`, `telefone`, `data_de_contratacao`, `cargo`, `salario`, `dia_de_pagamento`, `id_funcionario`) VALUES (\'Fulano de Tal10\', \'123 546 789-7\', \'123\', \'0\', \'2022-12-02\', \'123\', \'2022-12-05\', \'inútil\', \'2000\', \'5\', \'11\')')
 3 SYSTEMPATH\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `funcionarios` (`nome`, `rg`, `cpf`, `endereco`, `data_de_nascimento`, `telefone`, `data_de_contratacao`, `cargo`, `salario`, `dia_de_pagamento`, `id_funcionario`) VALUES (\'Fulano de Tal10\', \'123 546 789-7\', \'123\', \'0\', \'2022-12-02\', \'123\', \'2022-12-05\', \'inútil\', \'2000\', \'5\', \'11\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `funcionarios` (`nome`, `rg`, `cpf`, `endereco`, `data_de_nascimento`, `telefone`, `data_de_contratacao`, `cargo`, `salario`, `dia_de_pagamento`, `id_funcionario`) VALUES (:nome:, :rg:, :cpf:, :endereco:, :data_de_nascimento:, :telefone:, :data_de_contratacao:, :cargo:, :salario:, :dia_de_pagamento:, :id_funcionario:)', [...], false)
 5 SYSTEMPATH\Model.php(303): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(750): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(695): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\Funcionarios.php(63): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(925): App\Controllers\Funcionarios->store()
10 SYSTEMPATH\CodeIgniter.php(480): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Funcionarios))
11 SYSTEMPATH\CodeIgniter.php(349): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
ERROR - 2022-12-29 11:38:07 --> mysqli_sql_exception: Duplicate entry '11' for key 'PRIMARY' in C:\xampp\htdocs\CI4M2\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp\htdocs\CI4M2\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `fu...', 0)
#1 C:\xampp\htdocs\CI4M2\system\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `fu...')
#2 C:\xampp\htdocs\CI4M2\system\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `fu...')
#3 C:\xampp\htdocs\CI4M2\system\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `fu...', Array, false)
#4 C:\xampp\htdocs\CI4M2\system\Model.php(303): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\CI4M2\system\BaseModel.php(750): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\CI4M2\system\Model.php(695): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\CI4M2\app\Controllers\Funcionarios.php(63): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\CI4M2\system\CodeIgniter.php(925): App\Controllers\Funcionarios->store()
#9 C:\xampp\htdocs\CI4M2\system\CodeIgniter.php(480): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Funcionarios))
#10 C:\xampp\htdocs\CI4M2\system\CodeIgniter.php(349): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\CI4M2\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\CI4M2\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#13 {main}
CRITICAL - 2022-12-29 11:38:07 --> Duplicate entry '11' for key 'PRIMARY'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 292.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `funcionarios` (`nome`, `rg`, `cpf`, `endereco`, `data_de_nascimento`, `telefone`, `data_de_contratacao`, `cargo`, `salario`, `dia_de_pagamento`, `id_funcionario`) VALUES (\'Fulano de Tal10\', \'123 546 789-7\', \'123\', \'0\', \'2022-12-02\', \'123\', \'2022-12-05\', \'inútil\', \'2000\', \'5\', \'11\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `funcionarios` (`nome`, `rg`, `cpf`, `endereco`, `data_de_nascimento`, `telefone`, `data_de_contratacao`, `cargo`, `salario`, `dia_de_pagamento`, `id_funcionario`) VALUES (\'Fulano de Tal10\', \'123 546 789-7\', \'123\', \'0\', \'2022-12-02\', \'123\', \'2022-12-05\', \'inútil\', \'2000\', \'5\', \'11\')')
 3 SYSTEMPATH\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `funcionarios` (`nome`, `rg`, `cpf`, `endereco`, `data_de_nascimento`, `telefone`, `data_de_contratacao`, `cargo`, `salario`, `dia_de_pagamento`, `id_funcionario`) VALUES (\'Fulano de Tal10\', \'123 546 789-7\', \'123\', \'0\', \'2022-12-02\', \'123\', \'2022-12-05\', \'inútil\', \'2000\', \'5\', \'11\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `funcionarios` (`nome`, `rg`, `cpf`, `endereco`, `data_de_nascimento`, `telefone`, `data_de_contratacao`, `cargo`, `salario`, `dia_de_pagamento`, `id_funcionario`) VALUES (:nome:, :rg:, :cpf:, :endereco:, :data_de_nascimento:, :telefone:, :data_de_contratacao:, :cargo:, :salario:, :dia_de_pagamento:, :id_funcionario:)', [...], false)
 5 SYSTEMPATH\Model.php(303): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(750): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(695): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\Funcionarios.php(63): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(925): App\Controllers\Funcionarios->store()
10 SYSTEMPATH\CodeIgniter.php(480): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Funcionarios))
11 SYSTEMPATH\CodeIgniter.php(349): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
ERROR - 2022-12-29 11:38:14 --> mysqli_sql_exception: Duplicate entry '11' for key 'PRIMARY' in C:\xampp\htdocs\CI4M2\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp\htdocs\CI4M2\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `fu...', 0)
#1 C:\xampp\htdocs\CI4M2\system\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `fu...')
#2 C:\xampp\htdocs\CI4M2\system\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `fu...')
#3 C:\xampp\htdocs\CI4M2\system\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `fu...', Array, false)
#4 C:\xampp\htdocs\CI4M2\system\Model.php(303): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\CI4M2\system\BaseModel.php(750): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\CI4M2\system\Model.php(695): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\CI4M2\app\Controllers\Funcionarios.php(63): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\CI4M2\system\CodeIgniter.php(925): App\Controllers\Funcionarios->store()
#9 C:\xampp\htdocs\CI4M2\system\CodeIgniter.php(480): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Funcionarios))
#10 C:\xampp\htdocs\CI4M2\system\CodeIgniter.php(349): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\CI4M2\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\CI4M2\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#13 {main}
CRITICAL - 2022-12-29 11:38:14 --> Duplicate entry '11' for key 'PRIMARY'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 292.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `funcionarios` (`nome`, `rg`, `cpf`, `endereco`, `data_de_nascimento`, `telefone`, `data_de_contratacao`, `cargo`, `salario`, `dia_de_pagamento`, `id_funcionario`) VALUES (\'Fulano de Tal10\', \'123 546 789-7\', \'123\', \'0\', \'2022-12-02\', \'123\', \'2022-12-05\', \'inútil\', \'2000\', \'5\', \'11\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `funcionarios` (`nome`, `rg`, `cpf`, `endereco`, `data_de_nascimento`, `telefone`, `data_de_contratacao`, `cargo`, `salario`, `dia_de_pagamento`, `id_funcionario`) VALUES (\'Fulano de Tal10\', \'123 546 789-7\', \'123\', \'0\', \'2022-12-02\', \'123\', \'2022-12-05\', \'inútil\', \'2000\', \'5\', \'11\')')
 3 SYSTEMPATH\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `funcionarios` (`nome`, `rg`, `cpf`, `endereco`, `data_de_nascimento`, `telefone`, `data_de_contratacao`, `cargo`, `salario`, `dia_de_pagamento`, `id_funcionario`) VALUES (\'Fulano de Tal10\', \'123 546 789-7\', \'123\', \'0\', \'2022-12-02\', \'123\', \'2022-12-05\', \'inútil\', \'2000\', \'5\', \'11\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `funcionarios` (`nome`, `rg`, `cpf`, `endereco`, `data_de_nascimento`, `telefone`, `data_de_contratacao`, `cargo`, `salario`, `dia_de_pagamento`, `id_funcionario`) VALUES (:nome:, :rg:, :cpf:, :endereco:, :data_de_nascimento:, :telefone:, :data_de_contratacao:, :cargo:, :salario:, :dia_de_pagamento:, :id_funcionario:)', [...], false)
 5 SYSTEMPATH\Model.php(303): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(750): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(695): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\Funcionarios.php(63): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(925): App\Controllers\Funcionarios->store()
10 SYSTEMPATH\CodeIgniter.php(480): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Funcionarios))
11 SYSTEMPATH\CodeIgniter.php(349): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
ERROR - 2022-12-29 11:38:45 --> mysqli_sql_exception: Duplicate entry '11' for key 'PRIMARY' in C:\xampp\htdocs\CI4M2\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp\htdocs\CI4M2\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `fu...', 0)
#1 C:\xampp\htdocs\CI4M2\system\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `fu...')
#2 C:\xampp\htdocs\CI4M2\system\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `fu...')
#3 C:\xampp\htdocs\CI4M2\system\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `fu...', Array, false)
#4 C:\xampp\htdocs\CI4M2\system\Model.php(303): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\CI4M2\system\BaseModel.php(750): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\CI4M2\system\Model.php(695): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\CI4M2\app\Controllers\Funcionarios.php(63): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\CI4M2\system\CodeIgniter.php(925): App\Controllers\Funcionarios->store()
#9 C:\xampp\htdocs\CI4M2\system\CodeIgniter.php(480): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Funcionarios))
#10 C:\xampp\htdocs\CI4M2\system\CodeIgniter.php(349): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\CI4M2\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\CI4M2\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#13 {main}
CRITICAL - 2022-12-29 11:38:45 --> Duplicate entry '11' for key 'PRIMARY'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 292.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `funcionarios` (`nome`, `rg`, `cpf`, `endereco`, `data_de_nascimento`, `telefone`, `data_de_contratacao`, `cargo`, `salario`, `dia_de_pagamento`, `id_funcionario`) VALUES (\'Fulano de Tal11\', \'123 546 789-7\', \'123\', \'0\', \'2022-12-02\', \'123\', \'2022-12-05\', \'inútil\', \'2000\', \'5\', \'11\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `funcionarios` (`nome`, `rg`, `cpf`, `endereco`, `data_de_nascimento`, `telefone`, `data_de_contratacao`, `cargo`, `salario`, `dia_de_pagamento`, `id_funcionario`) VALUES (\'Fulano de Tal11\', \'123 546 789-7\', \'123\', \'0\', \'2022-12-02\', \'123\', \'2022-12-05\', \'inútil\', \'2000\', \'5\', \'11\')')
 3 SYSTEMPATH\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `funcionarios` (`nome`, `rg`, `cpf`, `endereco`, `data_de_nascimento`, `telefone`, `data_de_contratacao`, `cargo`, `salario`, `dia_de_pagamento`, `id_funcionario`) VALUES (\'Fulano de Tal11\', \'123 546 789-7\', \'123\', \'0\', \'2022-12-02\', \'123\', \'2022-12-05\', \'inútil\', \'2000\', \'5\', \'11\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `funcionarios` (`nome`, `rg`, `cpf`, `endereco`, `data_de_nascimento`, `telefone`, `data_de_contratacao`, `cargo`, `salario`, `dia_de_pagamento`, `id_funcionario`) VALUES (:nome:, :rg:, :cpf:, :endereco:, :data_de_nascimento:, :telefone:, :data_de_contratacao:, :cargo:, :salario:, :dia_de_pagamento:, :id_funcionario:)', [...], false)
 5 SYSTEMPATH\Model.php(303): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(750): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(695): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\Funcionarios.php(63): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(925): App\Controllers\Funcionarios->store()
10 SYSTEMPATH\CodeIgniter.php(480): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Funcionarios))
11 SYSTEMPATH\CodeIgniter.php(349): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2022-12-29 14:48:57 --> Undefined variable: funcionario
in APPPATH\Views\funcionarios\visualizar.php on line 29.
 1 APPPATH\Views\funcionarios\visualizar.php(29): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined variable: funcionario', 'APPPATH\\Views\\funcionarios\\visualizar.php', 29, [...])
 2 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\funcionarios\\visualizar.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1135): CodeIgniter\View\View->render('funcionarios/visualizar', [], true)
 5 APPPATH\Controllers\Funcionarios.php(53): view('funcionarios/visualizar', [...])
 6 SYSTEMPATH\CodeIgniter.php(925): App\Controllers\Funcionarios->visualizar('2')
 7 SYSTEMPATH\CodeIgniter.php(480): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Funcionarios))
 8 SYSTEMPATH\CodeIgniter.php(349): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2022-12-29 14:49:06 --> Undefined variable: funcionario
in APPPATH\Views\funcionarios\visualizar.php on line 29.
 1 APPPATH\Views\funcionarios\visualizar.php(29): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined variable: funcionario', 'APPPATH\\Views\\funcionarios\\visualizar.php', 29, [...])
 2 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\funcionarios\\visualizar.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1135): CodeIgniter\View\View->render('funcionarios/visualizar', [], true)
 5 APPPATH\Controllers\Funcionarios.php(53): view('funcionarios/visualizar', [...])
 6 SYSTEMPATH\CodeIgniter.php(925): App\Controllers\Funcionarios->visualizar('2')
 7 SYSTEMPATH\CodeIgniter.php(480): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Funcionarios))
 8 SYSTEMPATH\CodeIgniter.php(349): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2022-12-29 14:59:26 --> Too few arguments to function App\Controllers\Funcionarios::deletar(), 0 passed in C:\xampp\htdocs\CI4M2\system\CodeIgniter.php on line 925 and exactly 1 expected
in APPPATH\Controllers\Funcionarios.php on line 59.
 1 SYSTEMPATH\CodeIgniter.php(925): App\Controllers\Funcionarios->deletar()
 2 SYSTEMPATH\CodeIgniter.php(480): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Funcionarios))
 3 SYSTEMPATH\CodeIgniter.php(349): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2022-12-29 15:00:49 --> Too few arguments to function App\Controllers\Funcionarios::deletar(), 0 passed in C:\xampp\htdocs\CI4M2\system\CodeIgniter.php on line 925 and exactly 1 expected
in APPPATH\Controllers\Funcionarios.php on line 59.
 1 SYSTEMPATH\CodeIgniter.php(925): App\Controllers\Funcionarios->deletar()
 2 SYSTEMPATH\CodeIgniter.php(480): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Funcionarios))
 3 SYSTEMPATH\CodeIgniter.php(349): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2022-12-29 15:10:43 --> Too few arguments to function App\Controllers\Funcionarios::deletar(), 0 passed in C:\xampp\htdocs\CI4M2\system\CodeIgniter.php on line 925 and exactly 1 expected
in APPPATH\Controllers\Funcionarios.php on line 59.
 1 SYSTEMPATH\CodeIgniter.php(925): App\Controllers\Funcionarios->deletar()
 2 SYSTEMPATH\CodeIgniter.php(480): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Funcionarios))
 3 SYSTEMPATH\CodeIgniter.php(349): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2022-12-29 15:12:48 --> Too few arguments to function App\Controllers\Funcionarios::deletar(), 0 passed in C:\xampp\htdocs\CI4M2\system\CodeIgniter.php on line 925 and exactly 1 expected
in APPPATH\Controllers\Funcionarios.php on line 59.
 1 SYSTEMPATH\CodeIgniter.php(925): App\Controllers\Funcionarios->deletar()
 2 SYSTEMPATH\CodeIgniter.php(480): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Funcionarios))
 3 SYSTEMPATH\CodeIgniter.php(349): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2022-12-29 15:14:59 --> Undefined variable: id_funcionario
in APPPATH\Controllers\Funcionarios.php on line 64.
 1 APPPATH\Controllers\Funcionarios.php(64): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined variable: id_funcionario', 'APPPATH\\Controllers\\Funcionarios.php', 64, [])
 2 SYSTEMPATH\CodeIgniter.php(925): App\Controllers\Funcionarios->deletar()
 3 SYSTEMPATH\CodeIgniter.php(480): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Funcionarios))
 4 SYSTEMPATH\CodeIgniter.php(349): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
